docs/cugraph/source/tutorials/basic_cugraph.md

```diff
@@ -0,0 +1,28 @@
+## Required hardware/software
+
+CuGraph is part of [Rapids](https://docs.rapids.ai/user-guide)
```

Comment:
this sentence could be better worded and needs a period or it rolls into the next.  Also if you really want a newline then it needs to be stated "</br>"